src/Application.php

```diff
@@ -69,16 +40,11 @@ public function middleware($middlewareQueue)
             // and make an error page/response
             ->add(new ErrorHandlerMiddleware(null, Configure::read('Error')))
 
-            // Handle plugin/theme assets like CakePHP normally does.
-            ->add(new AssetMiddleware([
-                'cacheTime' => Configure::read('Asset.cacheTime')
-            ]))
-
             // Add routing middleware.
             // Routes collection cache enabled by default, to disable route caching
             // pass null as cacheConfig, example: `new RoutingMiddleware($this)`
             // you might want to disable this cache in case your routing is extremely simple
-            ->add(new RoutingMiddleware($this, '_cake_routes_'));
+            ->add(new RoutingMiddleware($this));
```

Comment:
Why do you want to remove routing cache? Same question for 
 https://github.com/phpbenchmarks/cake-php/pull/3, for REST API benchmark.

Reply:
Ok. I will make some tests, on Hello World with a single route maybe cache is not usefull, but with 500 routes for REST API it might be a huge gain.

Reply:
Why are 500 unnecessary routes setup for REST API to non existent actions?

Reply:
To represent a normal application, who will have our code + other routes. Some routers are really low, other are fast, so we add routes to take it into account.

Reply:
Okay, though I for symlex I couldn't find similar 500 routes, I just found 1 [here](https://github.com/phpbenchmarks/symlex/blob/symlex_4.1_rest-api/app/config/web.yml#L20). (I don't have any experience with symlex, so I could be looking in the wrong place)

Reply:
I can't find it too, so i've asked @lastzero to tell me if it's a miss.

Reply:
@ADmad Thanks for asking! We don't work with manual routes, we use callbacks (typically 2 for HTML/Twig and JSON/REST) and the service container for routing instead... so it's actual code and as efficient as that code is, typically O(1). Even with a million potential routes. This is because scalability and performance obviously is much better than a list of random, manual routes you need to go through. Also it's easier for developers to understand. We have that documented here: https://docs.symlex.org/en/latest/framework/routing/

What you could do to simulate 500 routes is adding 500 services/controllers to the service container instead. But as PHP should use a hash table, I guess it's also O(1) and makes no real difference.

Reply:
@lastzero IMO you should still add the "dummy" 500 routes like it's done for other frameworks to show how your framework isn't affect by those extra routes.

Reply:
Ok, so I should add 500 fake controller services based on the Symfony YAML file to see what happens? We don't support YAML files for routing otherwise and our Kernel is not designed to add individual routes.

Our approach is similar to what we did in Zend Framework 1 with `Zend_Controller_Router_Rewrite`. Nothing new or extraordinary.

Reply:
I think you should add 1 controller with 500 methods, or 500 controllers with 1 meathod each, to respect the 500 routes other frameworks have.

My bad, i haven't seen that was not present in Symlex REST API benchmark...

As soon as it will be added, i will relaunch benchmarks.

Reply:
@phpbenchmarks ASAP. I'll be traveling a lot until next week. I assume the results are about the same, but we should do this, just to be sure. Is the `/benchmark` route prefix important or can I use `/example/test-route-1`,... for the additional example routes? Output probably doesn't matter, so I can just display an empty page or "Hello World"?

Reply:
No problem. In that case I'll rename `BenchmarkController` to `HelloworldController` and `helloworldAction()` to `indexAction()`.

Reply:
@phpbenchmarks Thanks for the reminder! I have this on my todo. Can I get one or two more weeks before you get angry? Will do it ASAP.

Reply:
Yes i can wait 1 or 2 weeks, but not more. For now, Symlex is the first framework, but without an important feature. Maybe it will not change anything to add it, but that's not "fair" ;)

Reply:
@phpbenchmarks Looks like other `hello-world` examples actually don't have those 500 extra routes (note that this conversation is within the `cake-php_3.7_hello-world` branch): https://raw.githubusercontent.com/phpbenchmarks/symfony-common/symfony_4_hello-world/Resources/config/routing.yml

As for the REST example, the routes are actually handled by Symlex in that they all hit the same controller action and `test-route-*` is passed as string (not evaluated), so they are not unhandled and I didn't forget it. It's been a while since I wrote that code. But I see the intention to make routing slower / more complex for the purpose of the test. Working on it.

Reply:
@phpbenchmarks Done, but it looks like you revoked my write access. Trying to fork and send a pull request. Remember to update `composer.json` on your own to pull the changed code of the main and the common repo (I don't know how you like it tagged). It would also save time to keep the Docker config so that I don't have to restore it for future changes.